### PR TITLE
Dashboard schema: Review and mature timezone property

### PR DESCRIPTION
**Pull request by dprokop**

### CODE REVIEW

**Review by polibb — APPROVED** [review]
Yayyy 👏🏻

File: pkg/kinds/dashboard/dashboard_types_gen.go [hunk 1]
```diff
@@ -754,8 +745,8 @@ type Dashboard struct {
 		TimeOptions []string `json:"time_options"`
 	} `json:"timepicker,omitempty"`
 
-	// Timezone of dashboard,
-	Timezone *Timezone `json:"timezone,omitempty"`
+	// Timezone of dashboard. Accepts IANA TZDB zone ID or "browser" or "utc".
+	Timezone *interface{} `json:"timezone,omitempty"`
```
[the remark this answers is not in the document]
Comment (spinillos) [reply pkg/kinds/dashboard/dashboard_types_gen.go:749]: We can investigate that. Maybe removing `string` does the trick.

[the remark this answers is not in the document]
Comment (Clarity-89) [reply pkg/kinds/dashboard/dashboard_types_gen.go:749]: Wonder the reason for defining timezone as `string "browser" | "utc"`, doesn't it resolve to just string in the end? Or is it done to provide some autocomplete suggestions?

[the remark this answers is not in the document]
Comment (dprokop) [reply pkg/kinds/dashboard/dashboard_types_gen.go:749]: Ideally, we had an enum that abstracts all available timezones, but that's unnecessary overkill. The `browser` and `utc` values are Grafana's built-in values and they are complementary to IANA's zone IDs. Think we could define this just as a string, but the question then is, what should hold the information about `browser` and `utc` availability? Should that become part of a comment?

[the remark this answers is not in the document]
Comment (Clarity-89) [reply pkg/kinds/dashboard/dashboard_types_gen.go:749]: I'm not an expert on this but to me it makes sense to have it as string and describe `browser` and `utc` options in a comment.

[the remark this answers is not in the document]
Comment (dprokop) [reply pkg/kinds/dashboard/dashboard_types_gen.go:749]: @sdboyer - any thoughts on the above?

[the remark this answers is not in the document]
Comment (sdboyer) [reply pkg/kinds/dashboard/dashboard_types_gen.go]: yeah, i'm inclined to say that should be in the comment, which would reduce this to just `string`. (i haven't looked much at CUE's time package, but if it has timezone listings built in, that could be lovely here)

also, what is the behavior when the value is `""`? is it equivalent to `"browser"`? If so, that should be the default

[the remark this answers is not in the document]
Comment (dprokop) [reply pkg/kinds/dashboard/dashboard_types_gen.go]: yes,  "" defaults to browser.

**Pull request merged**